.github/workflows/build.yml

```diff
@@ -44,17 +53,28 @@ jobs:
       shell: bash
       run: git diff --exit-code || (echo '::error::Generated `tool_${{ matrix.generator }}` are out-of-date. Please run `cargo run -p tool_${{ matrix.generator }}`'; exit 1)
 
-  cargo_sys:
-    name: Check windows-sys
+  cargo_sys_stable:
+    name: Check windows-sys (stable)
     runs-on: windows-latest
     strategy:
       matrix:
-        rust: [1.46.0, stable, nightly]
+        rust: [1.46.0, stable]
     steps:
     - name: Checkout
       uses: actions/checkout@v2
     - name: Update toolchain
       run: rustup update --no-self-update ${{ matrix.rust }} && rustup default ${{ matrix.rust }}
+    - name: Run cargo check
+      run: cd crates/libs/sys && cargo check --features all_stable_features
```

Comment:
Since we have to generate the list either way, why don't we generate the list directly into the `build.yml` rather than exposing a *second* feature only to remove it later?

Reply:
I've made this change and pushed it to github; CI is just finishing up.  How do you typically handle things with this project?  Should I click "resolve conversation" as soon as the changes addressing the feedback have been pushed?

Reply:
Thanks, feel free to resolve.